viz/components/dust/slideshow/v1/index.tsx

```diff
@@ -283,14 +334,232 @@ export const Text = ({
   className,
 }: PropsWithChildren<{ className?: string }>) => (
   <p
-    className={cn("font-normal text-sm m-0", className)}
+    className={cn("font-normal text-lg m-0", className)}
     style={{ fontFamily: "var(--font-geist)" }}
   >
     {children}
   </p>
 );
 
-Slideshow.Slide = Slide;
-Slideshow.Title = Title;
-Slideshow.Heading = Heading;
-Slideshow.Text = Text;
+// Slide-level layout components.
+
+interface CoverProps {
+  className?: string;
+  isPreview?: boolean;
+  subtitle?: string;
+  subtitleClassName?: string;
+  title: string;
+  titleClassName?: string;
+}
+
+export const Cover = ({
+  className,
+  isPreview = false,
+  subtitle,
+  subtitleClassName,
+  title,
+  titleClassName,
+}: CoverProps) => (
+  <Slide variant="centered" className={className} isPreview={isPreview}>
+    <div className="h-full flex flex-col items-center justify-center text-center space-y-4">
+      <Title className={titleClassName}>{title}</Title>
+      {subtitle && (
+        <Text className={cn("opacity-80", subtitleClassName)}>{subtitle}</Text>
+      )}
+    </div>
+  </Slide>
+);
+Cover.displayName = "Slideshow.Slide.Cover";
+
+interface BulletsProps {
+  className?: string;
+  isPreview?: boolean;
+  items: string[];
+  title: string;
+  titleClassName?: string;
+}
+
+export const Bullets = ({
+  title,
+  items,
+  className,
+  titleClassName,
+  isPreview = false,
+}: BulletsProps) => (
+  <Slide variant="top" className={className} isPreview={isPreview}>
+    <div className="h-full flex flex-col space-y-6">
+      <Heading className={titleClassName}>{title}</Heading>
+      <ul className="space-y-4">
+        {items.map((item, index) => (
+          <li key={index} className="flex items-start">
+            <span className="text-2xl mr-4 mt-1 opacity-60">•</span>
+            <Text className="flex-1">{item}</Text>
+          </li>
+        ))}
+      </ul>
+    </div>
+  </Slide>
+);
+Bullets.displayName = "Slideshow.Slide.Bullets";
+
+type TwoChildren = [React.ReactNode, React.ReactNode];
+
+interface SplitProps {
+  children: TwoChildren;
```

Comment:
I don't really know how it works but this TwoChildren thing is a bit weird? We cannot have two different props?